python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -2310,6 +2311,44 @@ def testStatsAndLog( self ) :
 			driverEvents = [ x for x in stats if x["name"] == "ieCoreArnold:display:testBeauty" ]
 			self.assertEqual( driverEvents[0]["cat"], "driver_exr" )
 
+	def testMessageHandler( self ) :
+
+		RenderType = GafferScene.Private.IECoreScenePreview.Renderer.RenderType
+
+		for renderType, fileName, output in (
+			( RenderType.Batch, "", IECoreScene.Output( self.temporaryDirectory() + "/beauty.exr", "exr", "rgba" ) ),
+			( RenderType.Interactive, "", None ),
+			( RenderType.SceneDescription, self.temporaryDirectory() + "/test.ass" , None )
+		) :
+
+			with IECore.CapturingMessageHandler() as fallbackHandler :
+
+				handler = IECore.CapturingMessageHandler()
+
+				r = GafferScene.Private.IECoreScenePreview.Renderer.create(
+					"Arnold",
+					renderType,
+					fileName = fileName,
+					messageHandler = handler
+				)
+
+				r.option( "ai:console:info", IECore.BoolData( True ) )
```

Comment:
Does this mean that Arnold applies the console message settings to messages sent via the custom callback?

Reply:
Seems to. I think it's the difference between the transient stream and the file log stream. As they I think they coexist.